dss/operations/checkout.py

```diff
@@ -0,0 +1,183 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+import json
+import time
+import collections
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+from dss.storage.identifiers import DSS_BUNDLE_KEY_REGEX, VERSION_REGEX, UUID_REGEX, FILE_PREFIX
+from dss.storage.checkout.bundle import verify_checkout as bundle_checkout
+from dss.api.files import _verify_checkout as file_checkout
+from dss.storage.checkout import cache_flow
+
+logger = logging.getLogger(__name__)
+
+checkout = dispatch.target("checkout", help=__doc__)
+
+
+def verify_delete(handle, bucket, key):
+    logger.warning(f'attempting removal of key: {bucket}/{key}')
+    try:
+        handle.delete(bucket=bucket, key=key)
+        handle.get(bucket=bucket, key=key)
+    except BlobNotFoundError:
+        logger.warning(f'Success! unable to locate key {bucket}/{key} ')
```

Comment:
Nitpick: change the wording of this warning to "Successfully deleted key {bucket}/{key}"